### ISSUE TITLE
I tried to switch the camera from "back" to "front" but still got errors. 

### ISSUE THREAD

**Issue by DaoGatech** [Yalantis/CameraModule#10]
Right now I tried to use "intent" to recreate the activity with FRONT_CAMERA set to true so that I can switch the camera but it causes a lot of bugs. Do you have an easier way for this ?

**Comment by Kabir121** [comment]
Yep I can't take images through front camera for samsung devices. It says "camera.setParameters failed in android"

**Comment by Montsserrat** [comment]
In my case I resolved with :
     camera = Camera.open(Camera.CameraInfo.CAMERA_FACING_BACK);
You can change it for :
     camera = Camera.open(Camera.CameraInfo.CAMERA_FACING_FRONT);

 @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        boolean useFrontCamera = getArguments().getBoolean(FRONT_CAMERA, false);
        camera = Camera.open(Camera.CameraInfo.CAMERA_FACING_BACK);  // <-- THIS LINE
        //camera = getCameraInstance(useFrontCamera);
        if (camera == null) {
            return;
        }
        initScreenParams();
        parameters = camera.getParameters();
